test/jdk/javax/sql/resourceBundleTests/ValidateGetBundle.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import com.sun.rowset.JdbcRowSetResourceBundle;
+
+import java.util.Locale;
+import java.util.MissingResourceException;
+import java.util.PropertyResourceBundle;
+import java.util.ResourceBundle;
+
+/**
+ * @test
+ * @bug 8294989
+ * @summary Check JDBC RowSet resource bundle access
+ * @modules java.sql.rowset/com.sun.rowset
+ * @run main/othervm --add-opens java.sql.rowset/com.sun.rowset=ALL-UNNAMED ValidateGetBundle
+ */
+public class ValidateGetBundle{
+
+    // Resource bundle base name via a fully qualified class name
+    private static final String FULLY_QUALIFIED_CLASS_NAME = "com.sun.rowset.RowSetResourceBundle";
+    // Resource bundle base name via a path
+    private static final String PATH_TO_BUNDLE = "com/sun/rowset/RowSetResourceBundle";
+    // Default Locale
+    private static final Locale DEFAULT_LOCALE = Locale.getDefault();
```

Comment:
Is using the default locale OK? What if the test is run under some locale where JDBC does not provide the localized bundle?

Reply:
Switched default to constant US locale